README.md

```diff
@@ -233,6 +233,12 @@ tls_server_name = ""
 // The method to use for pinging external nodes. Defaults to "udp" but can
 // also be set to "socket" to use ICMP (which requires root privileges).
 ping_type = "udp"
+
+// how many failed checks trigger a state update to critical
+failures_before_critical = 1
+
+// how many successful checks trigger a state update to passing
+success_before_passing = 1
```

Comment:
It looks like these are currently defaulting to 0 in the code. These values can be set to 1 in `config.go` DefaultConfig()

Reply:
I have fixed this to be 0.

Reply:
Sounds good. 0 also aligns with the consul implementation, which is great